R/df_label.R

```diff
@@ -27,12 +37,15 @@
 #' )
 #'
 #' adsl <- xportr_df_label(adsl, metadata)
-xportr_df_label <- function(.df, metadata = NULL, domain = NULL, metacore = deprecated()) {
+xportr_df_label <- function(.df,
+                            metadata = NULL,
+                            domain = NULL,
```

Comment:
Even if not used in the function's body, it is documented in the man page

```suggestion
                            domain = NULL,
                            verbose = getOption("xportr.df_label_verbose", "none"),
```

Reply:
@averissimo I would agree that they should be consistent in their API, but I would say the API is driven by the functionality of the function itself. The functionality between the 'messaging' and 'non-messaging' functions are inconsistent (which should be updated in the next release) so I would say the API being different is ok.

Let me know what people think in the meeting. I'm happy to revert back if people disagree with the above.